app/controllers/libraries_controller.rb

```diff
@@ -16,17 +16,23 @@ def show
   # GET /libraries/new
   def new
     @library = Library.new
+    @library_categories = LibraryCategory.all
+
   end
 
   # GET /libraries/1/edit
   def edit
+    @library_categories = LibraryCategory.all
+    @library = Library.new
+
   end
 
   # POST /libraries
   # POST /libraries.json
   def create
+    @library_categories = LibraryCategory.all
```

Comment:
We could add a before_action, since it's the same sentence for create, edit, new.